src/components/Choice/Choice.scss

```diff
@@ -10,6 +10,10 @@
   align-items: center;
 }
 
+.ui__choice__input input {
```

Comment:
We have investigated the issue causing the checkboxes to go off-centre. We'll be implementing a fix for this soon. This particular CSS selector might affect other inputs so it would be best to leave this out for now.